lib/_tls_wrap.js

```diff
@@ -1315,6 +1315,9 @@ Server.prototype.setSecureContext = function(options) {
                                   .slice(0, 32);
   }
 
+  if (options.numTickets)
+    this.numTickets = options.numTickets;
```

Comment:
Ditto, plus it introduces a performance gotcha in that it creates two hidden classes: one with the property, one without. Always set the property.

Reply:
Hey @bnoordhuis could you please clarify how it creates two hidden classes: one with the property, one without?